pkg/odo/cli/cli.go

```diff
@@ -41,13 +42,16 @@ var (
  /  \__/    Find more information at https://odo.dev
  \__/`
 
-	odoExample = ktemplates.Examples(`  # Creating and deploying a Node.js project
-  git clone https://github.com/odo-devfiles/nodejs-ex && cd nodejs-ex
-  %[1]s create nodejs
-  %[1]s push
+	odoExample = ktemplates.Examples(`Initialize and take your pick from multiple languages or frameworks:
```

Comment:
Should we say initialize the component here? Something like:
```suggestion
	odoExample = ktemplates.Examples(`Initialize the component by taking your pick from multiple languages or frameworks:
```